frontend/src/modules/controllers/ValueController.tsx

```diff
@@ -1,3 +1,7 @@
+/**
+ * @summary Re-usable Wrapper container to control & display Paramaters value
```

Comment:
Where/how should this be used? (is it used at all in any actively-used code, e.g. the Set Value modal, or only in dead code?)

Reply:
It is used in active code It is the used to display value in Quickstart (Fio2 & Flow) & settings page (Display Tab) to show block of ValueController